light-client-verifier/src/operations/voting_power.rs

```diff
@@ -297,7 +302,12 @@ impl NonAbsentCommitVotes {
                 pair[0].validator_id(),
             ))
         } else {
-            Ok(Self { votes })
+            // As of protocol 0.38, 160 buffer is enough for chain identifier up
+            // to 44 bytes.
+            Ok(Self {
+                votes,
+                sign_bytes: Vec::with_capacity(160),
```

Comment:
Let's move the capacity to a constant, and move the comment there as well?

Reply:
Also, according to the CometBFT [docs](https://docs.cometbft.com/v0.37/core/using-cometbft#genesis) and [implementation](https://github.com/cometbft/cometbft/blob/v0.37.x/types/genesis.go?plain=1#L20), the ChainID can be of length up to 50 bytes, so let's account for that and increase the capacity to 166?